Dockerfile

```diff
@@ -8,6 +8,8 @@ ENV PYTHONUSERBASE=${VENV} PATH=${VENV}/bin:$PATH
 
 WORKDIR ${PROJECT_FOLDER}
 
+RUN python3 -m pip install --upgrade pip
```

Comment:
BTW in some other projects I added a `requirements.build.txt` to capture dependency versions like `pip`.

Reply:
this is good to know, I'll keep an eye out in future